TACC/TACC-Docs
# refactor: GH-36 delete manual permalinks for headings

## Overview

Delete manual permalinks for headings (cuz #37 will add automatic ones).

## Related

- coupled with #37
- mimicked by https://github.com/DesignSafe-CI/DS-User-Guide/pull/57

## Changes

- **deleted** links from headings (but kept `id`'s)

## Testing

1. Open http://0.0.0.0:8000/basics/mfa/.
2. Hover on "What is MFA?".
3. Verify it is not a link.

## UI

| before | after |
| - | - |
| <img width="1194" alt="before" src="https://github.com/user-attachments/assets/9cbad4ce-be05-4a1a-851f-a8f7d1044b91"> | <img width="1194" alt="after" src="https://github.com/user-attachments/assets/a5fa9f46-02ed-456d-ae61-2cc7263460f5"> |
